content/docs/introduction/billing-azure-marketplace.md

```diff
@@ -0,0 +1,61 @@
+2. Alternatively, go to the [Azure Marketplace](https://portal.azure.com/#view/Microsoft_Azure_Marketplace/MarketplaceOffersBlade/selectedMenuItemId/home) and search for **Neon Serverless Postgres**.
+3. Subscribe to the service.
+
+After subscribing, you will be directed to the [Create a Neon Resource](#create-a-neon-resource) page.
+
+## Create a Neon Resource
+
+1. On the **Create a Neon Resource** page, set the following values in the **Create Neon Resource** section.
+
+   | Property                   | Description                                                                                                                                                                                                                                                                                                        |
+   | -------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+   | **Subscription**           | From the drop-down, select your Azure subscription where you have Owner or Contributor access.                                                                                                                                                                                                                     |
+   | **Resource group**         | Specify whether you want to create a new resource group or use an existing one. A resource group is a container that holds related resources for an Azure solution. For more information, see [Azure Resource Group overview](https://learn.microsoft.com/en-us/azure/azure-resource-manager/management/overview). |
+   | **Resource Name**          | Enter the name for the Neon organization you want to create.                                                                                                                                                                                                                                                       |
+   | **Region**                 | Select a region to deploy your resource. Neon Serverless Postgres will let you choose a region while creating a project.                                                                                                                                                                                           |
+   | **Neon Organization name** | Corresponds to the name of your company, usually.                                                                                                                                                                                                                                                                  |
```

Comment:
Is this what we recommend? Neon "Organization" isn't currently in the Neon Docs Glossary page.

Reply:
I've expanded on that with a link to a glossary term.